EvidenceApi/V1/Gateways/DocumentsApiGateway.cs

```diff
@@ -62,9 +62,13 @@ public async Task<Claim> UpdateClaim(Guid id, ClaimUpdateRequest request)
 
         public async Task<Claim> GetClaimById(string id)
         {
+            var requestLimit = 20;
+            var throttler = new SemaphoreSlim(requestLimit);
```

Comment:
This will not throttle correctly as the `SemaphoreSlim` should be created outside of the loop.

Reply:
Addressed